applications/CoSimulationApplication/python_scripts/data_transfer_operators/copy_one_to_many.py

```diff
@@ -0,0 +1,61 @@
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_data_transfer_operator import CoSimulationDataTransferOperator
+
+def Create(settings):
+    return CopyOneToMany(settings)
+
+class CopyOneToMany(CoSimulationDataTransferOperator):
+    """DataTransferOperator to take one single value and set it to all values on another interface.
+    Used e.g. for FSI with SDof, where the SDof has one value and the fluid interface has many
+    """
+    def _ExecuteTransferData(self, from_solver_data, to_solver_data, transfer_options):
+        if not to_solver_data.IsDefinedOnThisRank():
+            return
+
+        data_value = 0.0
+        rank_for_broadcast = 0
+        if from_solver_data.IsDefinedOnThisRank():
+            data_value = from_solver_data.GetData()
+            if data_value.size == 1: # this is the rank that actually contains the value
+                data_value = data_value[0]
+                rank_for_broadcast = from_solver_data.GetModelPart().GetCommunicator().MyPID()
+
+        data_comm = to_solver_data.model_part.GetCommunicator().GetDataCommunicator()
+        data_value = data_comm.Broadcast(data_value, rank_for_broadcast)
```

Comment:
Is there a reason to use Broadcast instead of SumAll ? 
For my case, to map one node in structure(only on one rank) with multiple nodes in Fluid(nodes on every rank): Only one rank gets the right value from the from_solver side and can map this value to its nodes on the to_solver side.

But I think every rank has to get the value from this one node.

Reply:
it is already as you say
the rank that has the value (see check in line 19) broadcasts it to every other rank

In terms of your case:
one node on the structure (which can be on any rank, hence I need to save `rank_for_broadcast`) is being broadcasted to all ranks of the fluid.
Broadcast is one-to-all (see e.g. [here](https://mpitutorial.com/tutorials/mpi-broadcast-and-collective-communication/))